src/libcore/char.rs

```diff
@@ -136,230 +136,25 @@ pub fn from_digit(num: u32, radix: u32) -> Option<char> {
     }
 }
 
-/// Basic `char` manipulations.
-#[stable(feature = "rust1", since = "1.0.0")]
+// NB: the stabilization and documentation for this trait is in
+// unicode/u_char.rs, not here
```

Comment:
The `char` module in libunicode is what's actually exposed in the standard library, but the `str` module for the standard library is assembled in libcollections instead of libunicode, so I decided to leave the `u_str.rs` there for now (it's just a partial module). That being said I'd be fine reorganizing libunicode later on!